src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3436,6 +3436,18 @@ pub enum NamedPlan {
 }
 
 impl NamedPlan {
+    /// Return the [`NamedPlan`] for a given `path` if it exists.
+    pub fn of_path<'a>(value: &'a str) -> Option<Self> {
```

Comment:
If I do that I can no longer use `match value` for exhausting enumeration and need to run `find` or build some ad-hoc structure. Since the two methods are on top of each other and the set of stages is relatively stable (I don't expect it to change in the future) I would expect anybody that changes one method to also change the other.

If this is an issue I guess I can write a macro that generates the definitions of the two methods.

Reply:
Ok, then it's fine as it is.